lib/st/api/lims.pm

```diff
@@ -120,6 +120,12 @@ has 'driver_type' => (
                      );
 sub _build_driver_type {
   my $self = shift;
+  if($self->has_driver){
```

Comment:
1. Had thought it was too minor for Changes - but will do. Makes for unambiguous procedure....
2. Code change is "safe" in so far as passing a none  "st::api::lims::" class object will report the whole class name, passing a HASH will report HASH, and passing a scalar or undef will report "". Restricting by type, or provided methods, the driver attribute could well be useful - desk neighbour was considering putting a DBIC object in there....